tests/cli/samples/logs.yml

```diff
@@ -1,4 +1,5 @@
 - logs-all:
+  run: true
```

Comment:
changed to a `skip` field, so the default value is false and you can give true to skip said command